### PR TITLE
New package: Jackbox.Utility version 1.4.2+2

### LINKED ISSUE #175180: [Package Request]: Jackbox Utility
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://jackboxutility.com/
Publisher: Jackbox Utility
Package Name: Jackbox Utility
Description: An open-source application to launch Jackbox games.
Package Version: 1.4.2+2
Installer URL: https://github.com/AlexisL61/JackboxUtilityUpdater/releases/latest/download/JackboxUtility_Windows.zip

### PR DESCRIPTION
**Pull request by schembriaiden**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.6.0 :rocket:

- Resolves #175180

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/175456)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\Jackbox.Utility__DefaultSource\jackbox_patcher.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\jackbox.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 896.)

File: manifests/j/Jackbox/Utility/1.4.2+2/Jackbox.Utility.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: jackbox_patcher.exe
+  PortableCommandAlias: jackbox
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-08-16
+Installers:
```
Comment (stephengillie) [inline manifests/j/Jackbox/Utility/1.4.2+2/Jackbox.Utility.installer.yaml:16]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image

**Comment by vikingnope** [comment]
@stephengillie, I am getting the following errors for some DLLs:

image

image

image

These are all found within the downloaded zip, what needs to be done?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ##[error] Unsupported line end format Jackbox.Utility.installer.yaml

Hi @vikingnope,

It seems that a CRLF (`\r\n`) type of line ending has been used, instead of the required LF (`\n`) line ending. I've checked the file and can't find the `\r` character. Could you check for it?

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**